product_docs/docs/pgd/5/quickstart/further_explore_replication.mdx

```diff
@@ -11,44 +11,47 @@ With the cluster up and running, it's useful to run some basic checks to see how
 The following example shows one quick way to do this, but you must ensure that any testing you perform is appropriate for your use case.
 
 ### Preparation
```

Comment:
The structure in the following few sections was confusing because of the use of bullets instead of step numbers. In same cases, a bullet was used where it looks like it should have just been an introductory paragraph. Please check to make sure this maintains the integrity of what you intended.